### PR TITLE
fixes for MSVC build

### PR DESCRIPTION
**Pull request by ekoby**
update ziti-sdk-c to 0.23.1

### CODE REVIEW

File: programs/ziti-edge-tunnel/CMakeLists.txt [hunk 1]
```diff
@@ -14,6 +15,16 @@ if(CMAKE_SYSTEM_NAME STREQUAL Windows)
     set(NETIF_DRIVER_SOURCE netif_driver/windows/tun.c netif_driver/windows/tun.h)
 endif()
 
+if (MSVC)
+    message("using visual studio")
+    FetchContent_Declare(win-c
+            GIT_REPOSITORY https://github.com/ekoby/win-c.git
```
Comment (dovholuknf) [inline programs/ziti-edge-tunnel/CMakeLists.txt:21]: why `https://github.com/ekoby/win-c.git`  vs /netfoundry?

**Comment by dovholuknf** [comment]
x86 still doesn't build. x64 failed as well for me:
```
C:\git\github\openziti\ziti-tunneler-sdk-c\lib\ziti-tunnel\tunnel_tcp.c(123,9): error C2059: syntax error: ',' [C:\git\github\openziti\ziti-tunneler-sdk-c\build\x64\lib\ziti-tunnel\ziti-tunnel-sdk-
c.vcxproj]
C:\git\github\openziti\ziti-tunneler-sdk-c\lib\ziti-tunnel\tunnel_tcp.c(202,5): error C2059: syntax error: ',' [C:\git\github\openziti\ziti-tunneler-sdk-c\build\x64\lib\ziti-tunnel\ziti-tunnel-sdk-
c.vcxproj]
C:\git\github\openziti\ziti-tunneler-sdk-c\lib\ziti-tunnel\tunnel_tcp.c(208,5): error C2059: syntax error: ',' [C:\git\github\openziti\ziti-tunneler-sdk-c\build\x64\lib\ziti-tunnel\ziti-tunnel-sdk-
c.vcxproj]
C:\git\github\openziti\ziti-tunneler-sdk-c\lib\ziti-tunnel\tunnel_tcp.c(218,5): error C2059: syntax error: ',' [C:\git\github\openziti\ziti-tunneler-sdk-c\build\x64\lib\ziti-tunnel\ziti-tunnel-sdk-
c.vcxproj]
C:\git\github\openziti\ziti-tunneler-sdk-c\lib\ziti-tunnel\tunnel_tcp.c(229,5): error C2059: syntax error: ',' [C:\git\github\openziti\ziti-tunneler-sdk-c\build\x64\lib\ziti-tunnel\ziti-tunnel-sdk-
c.vcxproj]
````

File: programs/ziti-edge-tunnel/CMakeLists.txt (hunk 1 above)
Comment (ekoby) [reply programs/ziti-edge-tunnel/CMakeLists.txt:21]: argh, good catch, I was trying it our first and didn't want to pollute org if it didn't workout

**Review by dovholuknf — COMMENTED** [review]
fixed x86 - busted x64 for -G "Visual Studio 16 2019"

**Comment by dovholuknf** [comment]
oh nope - my bad - x86 still broken too. i just missed it in the output

**Comment by ekoby** [comment]
> oh nope - my bad - x86 still broken too. i just missed it in the output

any details you want to share?

**Comment by dovholuknf** [comment]
it was the same error. looking at the LOG_STATE macro it seems to fail to find __VA_ARGS++ in tunnel_tcp.c - line 13
image

**Review by dovholuknf — APPROVED** [review]

**Pull request merged**